go.mod

```diff
@@ -4,7 +4,7 @@ go 1.14
 
 require (
 	cloud.google.com/go v0.0.0-20170217213217-65216237311a
-	github.com/DataDog/datadog-go v0.0.0-20180822151419-281ae9f2d895
+	github.com/DataDog/datadog-go v3.7.2+incompatible
```

Comment:
I'd love to fix this, but it looks like we'll have to wait for https://github.com/DataDog/datadog-go/pull/150 so it isn't a blocker.

Reply:
As a customer, I've pinged the maintainers of this on their slack to get some movement on this PR.